ecommerce/baskets/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from customers.models import Customer
+from products.models import Product
+from core.models import BaseAbstractModel
+from baskets import enums
+
+
+class Basket(BaseAbstractModel):
+    customer = models.ForeignKey(Customer, verbose_name=_("Customer"),
```

Comment:
blank=True parametresini de eklersek, formlardan blank olarak gelmesine de izin vermis oluruz. Basket'i olan bir customer'in silinmesine izin vermek iyi bir yaklasim olmayabilir.